.github/workflows/static.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     steps:
       - uses: actions/checkout@master
       - name: PHPStan
-        uses: docker://jakzal/phpqa:php8.0
+        uses: docker://jakzal/phpqa:php8.2
```

Comment:
8.3 image is not ready yet but it should not be long. :)
https://github.com/jakzal/phpqa/pull/417